### PR TITLE
[10610]<fix>: fix TD-11023 TD-11024 TD-11079 TDinsight dashboard bugs

### PR DESCRIPTION
**Pull request by zitsen**
Include:
- [TD-11062]<fix>: fix names in TDinsight documents
- [TD-11079]<fix>: fix req_http_rate error in TDinsight
- [TD-11023]<fix>: fix Requests row loading time costs
- [TD-11024]<fix>: loading error while TDengine is not the default datasource

### CODE REVIEW

File: dashboards/TDinsight.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Then it's all done.
 
 The full page view for TDengine will like below.
 
-![display](tdengine-dashboard.png)
+![display](tdinsight.png)
```
Comment (sangshuduo) [inline dashboards/TDinsight.md:145]: TDinsight-full.png?

Comment (zitsen) [reply dashboards/TDinsight.md]: Yeah, fixed.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by afwerar — APPROVED** [review]
LGTM

**Pull request merged**